.github/workflows/golangci-lint.yaml

```diff
@@ -28,6 +28,6 @@ jobs:
           cache: false
 
       - name: golangci-lint
-        uses: golangci/golangci-lint-action@051d91933864810ecd5e2ea2cfd98f6a5bca5347 # v6.3.2
+        uses: golangci/golangci-lint-action@0adbc47a5910e47adb692df88187ec8c73c76778 # v6.4.0
         with:
-          version: v1.63 # Please keep this in sync with the Makefile!
+          version: v1.64.5 # Please keep this in sync with the Makefile!
```

Comment:
just keep `1.64` with that it will pick all patchs releases and we dont need to update everytime

Reply:
One thing I like about specifying the full patch version here is that it ensures parity between what we run locally with `make lint` and what CI is running, so we don't get surprises in CI